CHANGELOG.md

```diff
@@ -1,4 +1,231 @@
+
+### STATE BREAKING
+
+- [Provider](x/ccv/provider)
+  - Enable Opt In and Top N chains through gov proposals.
+    ([\#1587](https://github.com/cosmos/interchain-security/pull/1587))
+  - Assigning a key that is already assigned by the same validator will now be a no-op instead of throwing an error.
+    ([\#1732](https://github.com/cosmos/interchain-security/pull/1732))
+  - Adding the Partial Set Security feature cf. [ADR 015](https://cosmos.github.io/interchain-security/adrs/adr-015-partial-set-security).
+    ([\#1809](https://github.com/cosmos/interchain-security/pull/1809))
+  - Introduce power-shaping features for consumer chains. The features: (i) allow us to cap the total number of validators that can validate the consumer chain, (ii) set a cap on the maximum voting power (percentage-wise) a validator can have on a consumer chain, and (iii) introduce allowlist and denylists to restrict which validators are allowed or not to validate a consumer chain.
+    ([\#1830](https://github.com/cosmos/interchain-security/pull/1830))
+
+## v5.0.0
+
+❗ *The provider module should not be used with this version.*
+
+*May 9, 2024*
+
+### DEPENDENCIES
+
+- Bump [ibc-go](https://github.com/cosmos/ibc-go) to
+  [v8.1.x](https://github.com/cosmos/ibc-go/releases/tag/v8.1.0).
+  ([\#1698](https://github.com/cosmos/interchain-security/pull/1698))
+- Bump [CometBFT](https://github.com/cometbft/cometbft) to
+  [v0.38.4\5](https://github.com/cometbft/cometbft/releases/tag/v0.38.5).
+  ([\#1698](https://github.com/cosmos/interchain-security/pull/1698))
+- Bump [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) to
+[v0.50.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.4)
+([\#1698](https://github.com/cosmos/interchain-security/pull/1698))
+
+### FEATURES
+
+- [Consumer](x/ccv/consumer)
+  - Add consumer `MsgUpdateParams` from [cosmos-sdk](https://github.com/cosmos/cosmos-sdk).
+    ([\#1814](https://github.com/cosmos/interchain-security/pull/1814)).
+- [Provider](x/ccv/provider)
+  - Add provider `MsgUpdateParams` from [cosmos-sdk](https://github.com/cosmos/cosmos-sdk).
+    ([\#1698](https://github.com/cosmos/interchain-security/pull/1698)).
+
+### STATE BREAKING
+
+- Bump [ibc-go](https://github.com/cosmos/ibc-go) to
+  [v8.1.x](https://github.com/cosmos/ibc-go/releases/tag/v8.1.0).
+  ([\#1698](https://github.com/cosmos/interchain-security/pull/1698))
+- Bump [CometBFT](https://github.com/cometbft/cometbft) to
+  [v0.38.4\5](https://github.com/cometbft/cometbft/releases/tag/v0.38.5).
+  ([\#1698](https://github.com/cosmos/interchain-security/pull/1698))
+- Bump [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) to
+[v0.50.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.4)
+([\#1698](https://github.com/cosmos/interchain-security/pull/1698))
+- Revert `PutUnbondingOnHold` behavior to ICS@v1
+([\#1819](https://github.com/cosmos/interchain-security/pull/1819))
+## v4.2.0
```

Comment:
Why is this followed by `v4.3.0` in this file and not the reverse since we start with `v5.0.0`?

Reply:
It was probably mistakenly ordered during the merge. fixed by [0e3e05e1](https://github.com/cosmos/interchain-security/pull/2013/commits/0e3e05e1483808de8e70d80a22506d669e8ecad7).